WhateverGreen/kern_igfx.hpp

```diff
@@ -155,6 +155,54 @@ class IGFX {
 	 *  Framebuffer find / replace patches
 	 */
 	FramebufferPatch framebufferPatches[MaxFramebufferPatchCount] {};
+	
+	/**
+	 *  Framebuffer patch flags for first generation (Westmere).
+	 */
+	union FramebufferWestmerePatchFlags {
```

Comment:
Since we can't have multiple GEN drivers running simultaneously, consider merging this with FramebufferPatchFlags (e.g. as `bitsWestmere`) to conserve space.

Reply:
Resolved in 68ca19b.